LightBulb/ViewModels/Components/DashboardViewModel.cs

```diff
@@ -285,12 +288,63 @@ private void UpdateInstant()
         }
     }
 
-    private void UpdateConfiguration()
+    private const double _brightnessDefaultStep = 0.08;
+    private const double _temperatureDefaultStep = 30;
+
+    private double _brightnessMaxStep = _brightnessDefaultStep;
+    private double _temperatureMaxStep = _temperatureDefaultStep;
+
+    private ColorConfiguration _lastTarget;
+
+    private void UpdateConfiguration(double stepsPerSecond)
     {
-        var isSmooth = _settingsService.IsConfigurationSmoothingEnabled && !IsCyclePreviewEnabled;
+        if (CurrentConfiguration == TargetConfiguration)
+            return;
+
+        var isSmooth =
+            _settingsService.IsConfigurationSmoothingEnabled
+            && !IsCyclePreviewEnabled
+            && _settingsService.MaxSettingsTransitionDuration.TotalSeconds >= 0.1;
```

Comment:
Yes -- this fixes an edge case where, if the calculated transition duration is very small, it can lead to 0s or NaNs being produced by the calculation and fed into the smoothing function (which causes my monitors to go black, so it's near impossible to debug).